lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -20,7 +20,15 @@ def self.run(params)
           key = [file_name_hash, key].join('/')
         end
 
-        UI.user_error!("File already exists in S3 bucket #{bucket} at #{key}") if file_is_already_uploaded?(bucket, key)
+        if file_is_already_uploaded?(bucket, key)
+          message = "File already exists in S3 bucket #{bucket} at #{key}"
+          if params[:skip_if_exists]
+            UI.important("#{message}. Skipping upload.")
+          else
+            UI.user_error!(message)
+          end
+          return key
```

Comment:
What do you think of moving the `return` in the successful branch?

```suggestion
            UI.important("#{message}. Skipping upload.")
            return key
          else
            UI.user_error!(message)
          end
```

Since `UI.user_error!` will crash, it might be misleading to see the `return` after it in the code when looking at it top to bottom.